src/components/CodeView/index.spec.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import styles from './styles.module.scss';
+
+import CodeView from '.';
+
+describe(__filename, () => {
+  const render = ({
+    content = 'some content',
+    mimeType = 'mime/type',
+  } = {}) => {
+    return shallow(<CodeView mimeType={mimeType} content={content} />);
+  };
+
+  it('renders plain text code when mime type is not supported', () => {
+    const root = render();
```

Comment:
Instead of rely on the hidden default, this should explicitly set an unsupported mime type.